File: 0217. Contains Duplicate/Solution.py
class Solution:
    def containsDuplicate(self, nums):
        """
        type nums: List[int]
        rtype: bool
        """
        dic = {}
        for x in range(len(nums)):
            if nums[x] in dic:
                return True
            dic[nums[x]] = x
        return False

File: 0217. Contains Duplicate/Solution.cpp
#include <unordered_map>
#include <vector>

using std::unordered_map, std::vector;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int, int> mpp;
        for (int i = 0; i < nums.size(); i++)
        {
            if (mpp.find(nums[i]) != mpp.end())
                return true;
            mpp[nums[i]] = i;
        }
        return false;
    }
};